lib/src/room.dart

```diff
@@ -1282,7 +1284,8 @@ class Room {
       direction,
       from: prev_batch,
       limit: historyCount,
-      filter: jsonEncode(StateFilter(lazyLoadMembers: true).toJson()),
+      filter: jsonEncode(
```

Comment:
why did `lazyLoadMembers` change to false? Also I would just pass in the `Statefilter` as a parameter to the function instead of individual fields